### DISCUSSION TITLE
Interactions in Nested Popover

### DISCUSSION THREAD

**Discussion by NarendraKuruva** [adobe/react-spectrum#7520] [Q&A] [answered]
Hi all! 👋

I’ve been working with nested popovers and encountered a problem:

- When a nested popover contains an input that’s auto-focused, the parent popover closes.
- This makes it impossible to interact with the nested popover.
    - Click on buttons in nested list
    - Manually focus input

Here’s an example to demonstrate the issue:
[Codesandbox: Nested Popover with Input](https://codesandbox.io/p/sandbox/nested-popover-with-input-c74vzf?file=%2Fsrc%2FApp.js&workspaceId=ws_S2KoiQ3n9GNMw9BiK6Apdt)

I’d expect the parent popover to remain open while interacting with the nested one, but it seems like the focus logic is causing it to close.

Could you please take a look at this? Any guidance or suggestions would be much appreciated!

Tagging @devongovett and @snowystinger for input. Thanks so much! 😊

**Comment by NarendraKuruva** [comment]
@devongovett and @snowystinger can u once look into this?

**Reply by snowystinger** [reply, marked as answer]
Does https://github.com/adobe/react-spectrum/issues/4898#issuecomment-1674219912 work for you?
